src/views/Home/components/LostAndFoundCard/components/ActiveReports/index.tsx

```diff
@@ -33,17 +34,30 @@ const ActiveReports = () => {
   const isMobile = useMediaQuery('(max-width:375px)');
   const numReportsToDisplay = 4;
   const [snackbar, setSnackbar] = useState({ message: '', severity: undefined, open: false });
+  const [user, setUser] = useState({ AD_Username: '' });
   const navigate = useNavigate();
 
   const createSnackbar = useCallback((message, severity) => {
     setSnackbar({ message, severity, open: true });
   }, []);
 
+  useEffect(() => {
```

Comment:
Evan mentioned: "It should be almost trivial to call useUser in the components that call getMissingItemReportUser."  I believe there is a hook called useUser, that will get this information.  That will avoid having to call userService.getProfileInfo() every time.  Like he mentioned, calling this has to perform ~5 API requests, which is much slower than it needs to be to just get the username.  See if you can figure out using useUser instead of this, should be much more performant.